src/anomalib/models/fastflow/lightning_model.py

```diff
@@ -21,6 +21,7 @@ class Fastflow(AnomalyModule):
 
     Args:
         input_size (tuple[int, int]): Model input size.
+        image_size (tuple[int, int]): Original image size (needed in case of tiling).
```

Comment:
Would having both `image_size` and `input_size` in the model definition be confusing to the user? @ashwinvaidya17, any thoughts?

Reply:
Maybe we can just call it `original_image_size`?

Reply:
Also, not sure if I quite understood it. The only place we use the original_image_size is in anomaly_map_generator so that it can interpolate the flow maps. But why can't we pass the model input_size? Aren't the tiles generated from the input tensor that is in shape of model input_size? Also, won't the visualizer take care of re-scaling it further? Am I missing something?

Reply:
Input size is in case of tiling equal to tile size, because the fastflow model requires input size at the time of initialization. If we pass just tile size, we can't really know the size of anomaly maps, and if we pass just image size, we can't init the model as we don't know the tile size in the constructor. 

This is not a problem for other models as they don't require tile size in their constructors. And even if the visualizer handles this, the metrics don't as they expect the same size of GT and anomaly map.

On the other hand, I think this could be renamed to anomal_map_size, which would be better than image_size or original_image_size.

Reply:
Alright thanks for the explination. Yeah we can rename it to `anomaly_map_size`. Apart from this I have no other suggestions.